src/vivarium_csu_zenon/utilities.py

```diff
@@ -1,10 +1,47 @@
+import numpy as np
 import pandas as pd
+from scipy.stats import truncnorm
 
 from vivarium_public_health.risks.data_transformations import pivot_categorical
 
 from vivarium_csu_zenon import globals as project_globals
 
 
+class TruncnormParams:
+    def __init__(self, mean, sd, lower=0, upper=1):
+        self.a = (lower - mean) / sd if sd else mean
+        self.b = (upper - mean) / sd if sd else mean
+        self.loc = mean
+        self.scale = sd
+
+
+def sample_truncnorm_distribution(seed: int, mean: float, sd: float, lower: float = 0.0, upper: float = 1.0) -> float:
```

Comment:
I would just take an instance of the TruncnormParams here.  That's what they're packaged up for.